badge-maker/lib/badge-renderers.js

```diff
@@ -804,11 +804,13 @@ function forTheBadge({
   // there is no label. When `needsLabelRect` is true, render a label rect and a
   // message rect; when false, only a message rect.
   const hasLabel = Boolean(label.length)
+  const logoOnly = !hasLabel && !message
```

Comment:
shouldn't this be `const logoOnly = logo && !hasLabel && !message`

or maybe the variable should be called `noText` ?